modules/admin-ui/src/main/resources/public/org/opencastproject/adminui/languages/lang-en_US.json

```diff
@@ -872,15 +872,18 @@
           "WORKFLOWS": {
             "ACTIONS": "Actions",
             "DETAILS": "Details",
-            "DESCRIPTION": "",
-            "MORE_INFO": "More Information & Actions",
-            "ID": "ID",
+            "DESCRIPTION": "Description",
+            "MORE_INFO": "More Information",
+            "ID": "WIID",
             "TYPE": "Type",
             "TITLE": "Title",
-            "STATE": "State",
-            "STARTED": "Started",
+            "STATUS": "Status",
+            "STARTED": "Started At",
             "SERIES": "Series",
-            "SUBMITTED": "Submitted",
+            "SUBMITTED_AT": "Submitted At",
```

Comment:
Same here. Suggestion: `Submission date`

Reply:
I prefer "Submitted at". Easier to read / not as clumsy.

Note that "Submitted At" is displayed just next to its value which is a date - therefore, its meaning is obvious (see screenshots).

Reply:
If we're looking for shorter we could just use `Submitted`, which is clear enough with the date and time next to it.

Reply:
Changed to "Submitted"